src/Axiom/Set/TotalMap.agda

```diff
@@ -0,0 +1,40 @@
+{-# OPTIONS --safe --no-import-sorts #-}
+{-# OPTIONS -v allTactics:100 #-}
+
+open import Agda.Primitive renaming (Set to Type)
+open import Axiom.Set
+
+module Axiom.Set.TotalMap (th : Theory) where
+open Theory th
```

Comment:
Adding an empty line after `module` makes it easier to see. Also, do you need all those imports? Grouping them and sorting them alphabetically would also be nice.